WordPress/src/main/java/org/wordpress/android/ui/stats/refresh/lists/detail/PostAverageViewsPerDayUseCase.kt

```diff
@@ -24,7 +24,7 @@ import javax.inject.Inject
 import javax.inject.Named
 
 private const val BLOCK_ITEM_COUNT = 6
-private const val VIEW_ALL_ITEM_COUNT = 1000
+private const val VIEW_ALL_ITEM_COUNT = 300
```

Comment:
Just curious if there is a particular reason we don't have these use cases referencing a shared constant for this.

Reply:
I think the idea was that we could tweak the number of visible items per use case. I think it makes sense to use a common constant though (since it's all the same). I'll change it